ddtrace/contrib/kafka/patch.py

```diff
@@ -112,6 +116,7 @@ def traced_produce(func, instance, args, kwargs):
         span.set_tag(kafkax.PARTITION, partition)
         span.set_tag_str(kafkax.TOMBSTONE, str(value is None))
         span.set_tag(SPAN_MEASURED_KEY)
+        span.set_tag_str(kafkax.HOST_LIST, instance._dd_bootstrap_servers)
```

Comment:
Do we want to set this if the value is empty or None?

Reply:
Nope, wrong tracer for this (in NodeJS null tags are stripped) - fixed